distributions/otelcol-k8s/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.16 as certs
```

Comment:
Is there a reason this distro uses Alpine 3.16 whereas other distros use 3.19?

Reply:
Updated